Palace/Settings/TPPSettingsAccountsList.swift

```diff
@@ -171,8 +171,15 @@
   }
   
   private func updateList(withAccount account: Account) {
-    userAddedSecondaryAccounts.append(account.uuid)
+    if !userAddedSecondaryAccounts.contains(account.uuid) {
+      userAddedSecondaryAccounts.append(account.uuid)
+    }
     updateSettingsAccountList()
+    // Return from search screen to the list of libraries
+    updateNavBar()
+    tableView.reloadData()
+    navigationController?.popViewController(animated: false)
```

Comment:
this is now covered in line 187 below
```suggestion
```

Reply:
it is not, "Settings" screen remains on "Pick Your Library" screen:

https://user-images.githubusercontent.com/941531/130286360-e7823bcd-fcdd-49ce-bd8e-6b710e53d9cf.mov

Reply:
Ah, forgot to check iPad.